examples/org-workload-identity-provider/versions.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_version = ">= 0.15.0"
+
+  required_providers {
+    google = {
+      source                = "hashicorp/google"
+      version               = ">= 4.21.0"
+      configuration_aliases = [google.multiproject]
+    }
+    google-beta = {
+      source                = "hashicorp/google-beta"
+      version               = ">= 4.21.0"
+      configuration_aliases = [google-beta.multiproject]
+    }
+    sysdig = {
+      source  = "local/sysdiglabs/sysdig"
+      version = "~> 1.0.0"
```

Comment:
careful with these.
i guess you were testing local provider changes, so let's first publish those changes in the provider repo

Reply:
Yeah. I added it for testing it locally. I will remove it once we have a release.